src/test/java/com/google/api/tools/framework/snippet/SnippetTest.java

```diff
@@ -54,6 +58,28 @@ private SnippetSet createFromFile(String fileName) throws IOException  {
     }
   }
 
+  private SnippetSet createFromResource(String fileName) {
+    try {
+      final String root = getClass().getPackage().getName().replace('.', '/') + "/testdata";
+
+      InputSupplier supplier = new InputSupplier() {
+        @Override public Iterable<String> readInput(String snippetSetName) throws IOException {
+          try {
+            return Resources.readLines(Resources.getResource(root + "/" + snippetSetName), UTF_8);
+          } catch (Exception e) {
+            throw new IOException(e);
+          }
+        }
+      };
+
+      return snippets = SnippetSet.parse(supplier, fileName);
```

Comment:
To test the code change in the SnippetSet.java, can we just use the static method SnippetSet.resourceInputSupplier(root). Making this code:

SnippetSet.parse(SnippetSet.resourceInputSupplier(root), fileName);